src/services/auth-service.js

```diff
@@ -1,5 +1,6 @@
 import axios from "axios";
-const BASE_URL = process.env.REACT_APP_TUIT_SERVICE_URL;
+//const BASE_URL = process.env.REACT_APP_TUIT_SERVICE_URL;
+ const BASE_URL = "http://localhost:4000/api" | "http://localhost:4003/api";
```

Comment:
Please use the env variable instead of hardcoded strings!

Reply:
For some reason, the env variables weren't working when I was trying to log in so that's why I put hardcoded strings. But I'll change it back. Sorry!